tests/main/test_integration.py

```diff
@@ -745,24 +802,27 @@ def test_app_shutdown__in_worst_case_while_recording_is_running(
             f"{get_api_endpoint()}start_recording?barcode={expected_barcode}&is_hardware_test_recording=False"
         )
         assert response.status_code == 200
-
         assert system_state_eventually_equals(RECORDING_STATE, 5) is True
 
         time.sleep(3)  # Tanner (6/15/20): This allows data to be written to files
 
+        # Tanner (12/29/20): Shutdown now that data is being acquired and actively written to files, which means each subprocess is doing something
         response = requests.get(f"{get_api_endpoint()}shutdown")
         assert response.status_code == 200
         # TODO (Eli 12/9/20): have /shutdown wait to return until the other processes have been stopped, or have a separate route to shut down other processes (that also waits)
```

Comment:
ignore this `TODO` for now.  that's going to involve some more complicated interaction with the front-end that I don't want to dive into just yet